elixir/apps/api/mix.exs

```diff
@@ -79,4 +77,9 @@ defmodule API.MixProject do
       test: ["ecto.create --quiet", "ecto.migrate", "test"]
     ]
   end
+
+  defp version do
+    sha = System.get_env("GIT_SHA", "deadbeef")
```

Comment:
Should there be a `|> String.trim()` here to keep things as close to what was as possible?

Reply:
👍